acceptance/tests/restart_host_run_puppet.rb

```diff
@@ -22,8 +22,14 @@
   step "restart each agent" do
     applicable_agents.each do |agent|
       agent.reboot
+      # BKR-812
+      sleep 1 if !agent.up?
```

Comment:
it worked locally.  test repeatedly fails without it and repeatedly passes with it.  but i can do some fall-back type stuff.  we just need to get CI green so we can ship PA 1.5.1